Apps/Database/src/Models/AgentAudit.cs

```diff
@@ -49,19 +49,26 @@ public class DependentAudit : AuditableEntity
         public string AgentUsername { get; set; } = string.Empty;
 
         /// <summary>
-        /// Gets or sets the protected reason.
+        /// Gets or sets the reason.
         /// </summary>
         [Required]
         [MaxLength(500)]
-        public string ProtectedReason { get; set; } = string.Empty;
+        public string Reason { get; set; } = string.Empty;
 
         /// <summary>
-        /// Gets or sets a value representing the type of dependent audit operation.
-        /// The value is one of <see cref="DependentAuditOperation"/>.
+        /// Gets or sets a value representing the type of audit operation.
+        /// The value is one of <see cref="AuditOperation"/>.
         /// </summary>
         [Required]
         [MaxLength(10)]
-        public DependentAuditOperation OperationCode { get; set; } = DependentAuditOperation.Unprotect;
+        public AuditOperation OperationCode { get; set; } = AuditOperation.UnprotectDependent;
+
+        /// <summary>
+        /// Gets or sets the group.
+        /// </summary>
+        [Required]
+        [MaxLength(10)]
```

Comment:
I think this need to be more than 10 - go with at least 50.

Reply:
@ytqsl A bug was going to be created to change all code tables regarding column length of that column.

Reply:
As this is a new column, wouldn't it be better to increase the length before merging to dev?

Reply:
Changed.